2-mandatory/4-stocks.js

```diff
@@ -48,7 +56,11 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+    let priceChange = [];
+      for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+        priceChange.push(Math.round((closingPricesForAllStocks[i][closingPricesForAllStocks[i].length-1] - closingPricesForAllStocks[i][0]) * 100) / 100);
```

Comment:
This is good, but it would be a good idea to split this into multiple lines to improve the readability of the code.